modules/distribution/carbon-home/conf/dashboard/deployment.yaml

```diff
@@ -366,6 +366,23 @@ wso2.datasources:
         validationTimeout: 30000
         isAutoCommit: false
 
+  - name: EI_ANALYTICS
+    description: "The datasource used for EI Analytics dashboard feature"
+    jndiConfig:
+      name: jdbc/EI_ANALYTICS
+    definition:
+      type: RDBMS
+      configuration:
+        jdbcUrl: 'jdbc:h2:${sys:carbon.home}/wso2/dashboard/database/EI_ANALYTICS;AUTO_SERVER=TRUE'
```

Comment:
Lets create the database in worker profile as a standard. jdbc:h2:${sys:carbon.home}/wso2/worker/database/EI_ANALYTICS;AUTO_SERVER=TRUE

Reply:
Fixed in 97933ae0a8e13300895900efb121821109608a10